maestro-ios-xctest-runner/maestro-driver-iosUITests/Routes/Handlers/RunningAppRouteHandler.swift

```diff
@@ -9,8 +9,12 @@ struct RunningAppRouteHandler: HTTPHandler {
         category: String(describing: Self.self)
     )
     
-    private static let springboardBundleId = "com.apple.springboard"
-    
+    #if os(tvOS)
+    private static let homescreenBundleId = "com.apple.HeadBoard"
```

Comment:
On AppleTV the home screen is made up of a process called HeadBoard (the UI and cards) which is layered over another process called PineBoard (the top-right dropdown and background image). We can use the HeadBoard bundleId to allow interacting with cards inside of Studio and we can use PineBoard to get the screen dimensions